v2/lockrenewer.go

```diff
@@ -34,11 +39,16 @@ type LockRenewalOptions struct {
 // NewRenewLockHandler returns a middleware handler that will renew the lock on the message at the specified interval.
 func NewRenewLockHandler(options *LockRenewalOptions, handler Handler) HandlerFunc {
 	interval := 10 * time.Second
+	lockRenewalTimeout := interval
 	cancelMessageContextOnStop := true
 	metricRecorder := processor.Metric
 	if options != nil {
 		if options.Interval != nil {
 			interval = *options.Interval
+			lockRenewalTimeout = interval
```

Comment:
I don't mind this but I don't think making the timeout relative to the interval makes much sense.
to reduce network bandwidth and increase throughput, you increase that interval to be less chatty.
so, say 1min lock 20s interval.
I would still want just a few seconds of timeout. not 20s.

Reply:
I think this is more of a default if application code doesn't set anything in its options? Since we know the interval has to be less than the lock duration, this default will allow the lock renewer exit before the lock expires.

Reply:
edit: changed to default to 5s if not set or 0